packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -52,34 +40,17 @@ const BIContractActivationWindow = ({
     return queryParams.get('connection_deleted') === 'true'
   }
 
+  const afterOAuthWindow = async result => {
```

Comment:
you don't use result. Is it intended?

Reply:
I use result.data.finalLocation. I updated the jsdoc